models/_sector/depin/geodnet/geodnet_revenue.sql

```diff
@@ -0,0 +1,127 @@
+{{ config(
+    schema = 'geodnet'
```

Comment:
```suggestion
    schema = 'geodnet_polygon'
```

Reply:
this spell contains hardcoded `polygon` filters, we should follow our standards for schema names in that case